bifrost-fastify/index.ts

```diff
@@ -49,8 +48,11 @@ async function replyWithPage(
     return reply.code(404).type("text/html").send("Not Found");
   }
 
-  const { body, statusCode, contentType } = httpResponse;
-  return reply.status(statusCode).type(contentType).send(body);
+  const { body, statusCode, headers } = httpResponse;
+  return reply
+    .status(statusCode)
+    .headers(Object.fromEntries(headers))
```

Comment:
vps returns any number of heads now, not just content type. this enables the trailing slash redirect built-in to vps